src/main/webapp/app/pages/annotationPage/CancerTypeViewTable.tsx

```diff
@@ -27,35 +30,50 @@ export interface ITableRow {
   description?: string;
 }
 
+function inCompactView(store: WindowStore) {
+  return !store.isLargeScreen;
+}
+
+function geDrugAssociationCellText(
+  hugoSymbol: string,
+  alterations: string,
+  cancerTypes: string
+) {
+  const altNameIncludesHugoSymbol = alterations.includes(hugoSymbol);
+  return `Associated with ${
+    altNameIncludesHugoSymbol ? '' : hugoSymbol
+  } ${alterations} in ${cancerTypes}`;
+}
+
 const CompactDrugCell: React.FunctionComponent<{
   hugoSymbol: string;
   alterations: string;
   cancerTypes: string;
   drugs: string;
 }> = props => {
-  const altNameIncludesHugoSymbol = props.alterations.includes(
-    props.hugoSymbol
-  );
   return (
     <div style={{ minWidth: 300 }}>
       <div>{props.drugs}</div>
       <div className={'text-secondary'}>
-        Associated with{' '}
-        {altNameIncludesHugoSymbol ? '' : `${props.hugoSymbol} `}
-        {props.alterations}
-      </div>
-      <div className={'text-secondary'}>
-        in {props.cancerTypes.toLowerCase()}
+        {geDrugAssociationCellText(
+          props.hugoSymbol,
+          props.alterations,
+          props.cancerTypes
```

Comment:
No longer converting to lowercase?

Reply:
Yeah, some cancer type names include genetic biomarkers. They are incorrect to be converted to lowercase. I'm thinking to make it simple without any conversion, similar to what we show on the title. If later user requests the name to be converted to lowercase, then we will spend more time fixing.